cmd/prometheus/main.go

```diff
@@ -621,11 +627,7 @@ func main() {
 	noStepSubqueryInterval := &safePromQLNoStepSubqueryInterval{}
 	noStepSubqueryInterval.Set(config.DefaultGlobalConfig.EvaluationInterval)
 
-	// Above level 6, the k8s client would log bearer tokens in clear-text.
-	klog.ClampLevel(6)
```

Comment:
Seem to have lost this line.

Reply:
So `ClampLevel()` isn't a native klog function, but rather a function from the library prometheus replaces klog with:
https://github.com/prometheus/prometheus/blob/main/go.mod#L210-L213
https://pkg.go.dev/github.com/simonpasquier/klog-gokit#ClampLevel
https://pkg.go.dev/github.com/simonpasquier/klog-gokit/v3#ClampLevel

A similar change was made to the prometheus operator when they switched to slog:
https://github.com/prometheus-operator/prometheus-operator/pull/6902/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6L147
https://github.com/prometheus-operator/prometheus-operator/pull/6902/files#diff-3a86c514d2d3f52dc7235e0444e3a2c54960795d7e73cecb131b876ce306442bL67-L69

To the best of my current understanding, this is intentional